pages/common/pio-project.md

```diff
@@ -0,0 +1,24 @@
+# pio project
+
+> Tool to manage PlatformIO projects.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/project/index.html>.
+
+- Initialize a new PlatformIO based project:
```

Comment:
Why the `based` here?
```suggestion
- Initialize a new PlatformIO project:
```

Reply:
There are tools built upon platformio, which use the same project structure. But I guess, that's not important here.